src/modules/job-manager/plugins/perilog.c

```diff
@@ -69,7 +70,9 @@ extern char **environ;
 static struct perilog_conf {
     double prolog_kill_timeout;/*  Time between SIGTERM/SIGKILL for prolog  */
     flux_cmd_t *prolog_cmd;    /*  Configured prolog command                */
+    bool prolog_per_rank;      /*  Execute prolog on each job rank          */
```

Comment:
Commit message "per-task" should be "per-rank"? Sorry if I'm misreading that, I just don't see any references to tasks in the code.